coredb-operator/yaml/crd.yaml

```diff
@@ -31,12 +31,15 @@ spec:
                   type: string
                 type: array
               image:
-                default: quay.io/coredb/postgres:2023.01.24
+                default: quay.io/coredb/postgres:c03124e
```

Comment:
It was failing because the default user of this container is postgres, so I fixed by updating the init container to run as root. It did not fail locally because I forgot to locally apply the CRD again.